examples/main.tf

```diff
@@ -25,10 +25,17 @@ resource "buildkite_agent_token" "fleet" {
   description = "token used by build fleet"
 }
 
+resource "buildkite_cluster_queue" "queue1" {
+  cluster_id = "Q2x1c3Rlci0tLTMzMDc0ZDhiLTM4MjctNDRkNC05YTQ3LTkwN2E2NWZjODViNg=="
```

Comment:
I was wondering if we want to expose the GraphQL IDs for our own resources, but we have them hard coded in tests anyway so guess it's not a huge concern.